### DISCUSSION TITLE
How to save pictures as dynamic

### DISCUSSION THREAD

**Discussion by OrionPotter** [jgraph/drawio#4146] [General]
Now I drew an architecture diagram, which contains dotted arrows. In the properties, I set flow for it. Now I save it as a picture format, png or svg, but neither the dynamics of the arrows can be saved. Is there any good way? Save this image as a dynamic image

**Comment by alderg** [comment]
Flow animation support was added to SVG export in version 21.8.2.

**Reply by OrionPotter** [reply]
That's Great
